### PR TITLE
make label_image.py v2 compatible

### PR DESCRIPTION
**Pull request by freedomtan**
Update `tensorflow/examples/label_image/label_image.py` to make it V2 compatible.

### CODE REVIEW

**Comment by gbaned** [comment]
@freedomtan Could you please resolve the conflicts? Thanks!

File: tensorflow/examples/label_image/label_image.py [hunk 1]
```diff
@@ -40,33 +40,37 @@ def read_tensor_from_image_file(file_name,
...
         file_reader, channels=3, name="jpeg_reader")
-  float_caster = tf.cast(image_reader, tf.float32)
-  dims_expander = tf.expand_dims(float_caster, 0)
-  resized = tf.image.resize_bilinear(dims_expander, [input_height, input_width])
-  normalized = tf.divide(tf.subtract(resized, [input_mean]), [input_std])
-  sess = tf.compat.v1.Session()
-  result = sess.run(normalized)
+    float_caster = tf.cast(image_reader, tf.float32)
+    dims_expander = tf.expand_dims(float_caster, 0)
+    resized = tf.image.resize(dims_expander, [input_height, input_width], method=tf.image.ResizeMethod.BILINEAR)
```
Comment (caisq) [inline tensorflow/examples/label_image/label_image.py]: style nit: Respect the 80-char line width limit.

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/examples/label_image/label_image.py (hunk 1 above)
Comment (freedomtan) [reply tensorflow/examples/label_image/label_image.py]: oops. thanks

**Review by caisq — DISMISSED** [review]

**Comment by gbaned** [comment]
@freedomtan Could you please check failed build errors? Thanks!

**Comment by freedomtan** [comment]
@gbaned it seems none of them are relevant. Maybe the master branch happened to have problems. E.g., in the [Ubuntu Python3 PIP](https://source.cloud.google.com/results/invocations/1f017dbe-c933-463f-a739-c6fd1291ae9d/log) log, the problem is in LLVM related code.

**Comment by gbaned** [comment]
@freedomtan Thank you. Can you please check Ubuntu Sanity failures?

**Comment by freedomtan** [comment]
I don't know how to check it. I can see logs of Ubuntu Python3 PIP, Ubuntu contrib, and Windows Bazel GPU. But not Ubuntu Sanity and Windows Bazel. See the figure below

image

**Comment by gbaned** [comment]
@freedomtan Sorry for the late response. Can you please check Ubuntu Sanity failures now? Thank you.

**Comment by freedomtan** [comment]
@gbaned Thanks. Yes, I saw them. Fixed pylint Cs.

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
@caisq Can you please take a look on this PR? Thanks!

**Comment by caisq** [comment]
Adding @MarkDaoust to reviewers. He's looking into the examples directory recently.

**Comment by MarkDaoust** [comment]
@freedomtan, thanks for taking the time to put together this PR.

But even with these updates, this is not the sort of code we want to encourage:

    1. Nobody should be using raw GraphDef's anymore.
    2. I'd rather not have any examples directly using Session.

How did you get pointed to this example? someone should update that link.

Modern examples of how to do something like this is included in: 

https://github.com/tensorflow/docs/blob/master/site/en/tutorials/images/transfer_learning_with_hub.ipynb
https://github.com/tensorflow/docs/blob/master/site/en/guide/saved_model.ipynb

I'm working on deleting this examples/ directory. Please submit future PRs to https://github.com/tensorflow/examples

**Comment by freedomtan** [comment]
@MarkDaoust I wrote it originally :)

**Comment by MarkDaoust** [comment]
> I wrote it originally :)

😲

I do appreciate you taking the time to come back and update it.
And I hope you understand that I mean no disrespect to you or your work.

**Comment by nanshihui** [comment]
i use  label_image.py to predict a lot of picture .however the memory leak. anyone find it ?

**Comment by MarkDaoust** [comment]
Hi @nanshihui please see the examples I linked to previously for a modern way to do this.

**Comment by nanshihui** [comment]
i turn hub.Module params    trainable to True,
it return 
```
tensorflow.python.framework.errors_impl.InvalidArgumentError: You must feed a value for Placeholder  with dtype float and shape [?,299,299,3]

```
do you have any ways? @MarkDaoust

**Pull request closed without merging**